lib/ruby2d/window.rb

```diff
@@ -614,7 +621,12 @@ def render_callback
 
     # Show the window
     def show
+      if @@open_window
+        raise Error, "Window#show called multiple times, Ruby2D only supports a single open window"
+      end
+
       ext_show
+      @@open_window = true
```

Comment:
Setting `@@open_window` would have to be done before `ext_show`, because that enters into the infinite main loop.

Reply:
Thanks, fixed in ee3a9c547a2d11a724ff7b819cedb254a0911b73